package.json

```diff
@@ -14,6 +14,7 @@
     "node-sass": "4.14.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-router-dom": "^5.2.0",
```

Comment:
Quick question, I thought we already have this dependency. Or at first it wasn't listed on your **PR**? 🤔 @kimvcp

Reply:
In my PR, this `react-router-dom` was installed only the `@types` scope for Typescript. However, we could have it together with the `@types` one.

Reply:
I just found that it is the best practice to have both `react-router-dom` and `@types/react-router-dom` installed.
Ref: https://www.sitepoint.com/react-with-typescript-best-practices/